src/main/charts/bitbucket/Chart.yaml

```diff
@@ -10,7 +10,9 @@ keywords:
   - Bitbucket Server
   - Bitbucket Data Center
   - Atlassian
-home: https://github.com/atlassian-labs/data-center-helm-charts
+home: https://www.atlassian.com/software/bitbucket
+icon: https://wac-cdn.atlassian.com/dam/jcr:bf39fc40-3871-491f-98e3-fb2293f57a00/bitbucket-icon-gradient-blue.svg?cdnVersion=696
```

Comment:
Is this URL stable?

Reply:
I copied them from our docker image READMEs.md and they are stable - For example in Confluence the last update was on `2019-12-01`.